components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/Types.kt

```diff
@@ -104,25 +107,25 @@ fun Profile.into(): mozilla.components.concept.sync.Profile {
     )
 }
 
-internal fun Device.Type.into(): DeviceType {
```

Comment:
The bindings generated doesn't support these nested namespace things yet, so e.g. `Device.Type` is now `DeviceType`.